.github/workflows/main.yml

```diff
@@ -5,7 +5,9 @@ on:
     branches:
       - main
       - master
+      - stable
       - Version-v*
+      - release/v*
       - trigger-ci*
   pull_request:
```

Comment:
In a recent PR, a new `branches-ignore` property was added here. We should update this branch, and add `stable` alongside it.